src/components/_channel/ChannelWithVideos/ChannelWithVideos.tsx

```diff
@@ -86,14 +97,7 @@ export const ChannelWithVideos: FC<ChannelWithVideosProps> = ({ channelId }) =>
           {isLoading ? (
             <SkeletonLoader width="90px" height="40px" />
           ) : (
-            <FollowButton
-              variant="secondary"
-              size={'medium'}
-              onClick={(e) => {
-                e.preventDefault()
-                toggleFollowing()
-              }}
-            >
+            <FollowButton variant="secondary" size={'medium'} onClick={handleFollowClick}>
```

Comment:
Again, probably not your change 😄 , but we could do some cleanup 
```suggestion
            <FollowButton variant="secondary" size="medium" onClick={handleFollowClick}>
```